### DISCUSSION TITLE
aplus-ui/components/ap-descriptions/

### DISCUSSION THREAD

**Discussion by giscus** [ollieLiu1988/aplus-ui-comments#9] [General]
# aplus-ui/components/ap-descriptions/

Aplus basic doc site.

https://aplus.aplnk.com/aplus-ui/components/ap-descriptions/

**Comment by ollieLiu1988** [comment]
ApDescriptions 描述列表组件,全员通知测试

**Comment by Ashe-reset** [comment]
value如果是文本类型，可否支持一下ellipsis？@Ezreal

**Reply by MyMagicCode** [reply]
已支持，提供format进行数据转换。
